src/svgutils/transform.py

```diff
@@ -13,6 +13,8 @@
 XLINK = "{%s}" % XLINK_NAMESPACE
 NSMAP = {None: SVG_NAMESPACE, "xlink": XLINK_NAMESPACE}
 
+from svgutils.common import Unit
```

Comment:
This import line seems to be too far down: it would result in definition of constants in between existing imports and this new import. I'd expect the new import e.g. as the last import between current line 8 and 10, instead.

Reply:
done